advice/initial-meeting-with-Sion.md

```diff
@@ -0,0 +1,115 @@
++ Having a common vision. Keeping that strong. Making it explicit. Writing it down - having a manifesto/vision. That's your membership orientation document.
++ Pitfalls of being in a trading enterprise will arise naturally and you will learn that as you go
++ Business administration e.g. bookkeeping, maintaining register of members, running your governence stuff (making sure everything is recorded well). Do you have people who specialise in business admin or do you rotate those tasks? How do you service those "rather boring" aspects, that aren't core to what you're producing?
++ Political disagreements within the group. Because the enterprise was growing and we had work, everything looked like it was going well, we didn't address it. We became inefficient and it affected the quality of the work. We tried to solve it by bringing more people in.
++ Open book management. Making finances available to people in a way that they can understand. If you don't understand it, you can't be in control of it. When co-ops get into difficulties, it's when things aren't transparent. Maybe somebody whose role is dealing with financial information decides to sit on something and try and solve it, thinking "they don't need to know this". When you veer away from open, honest, solidarity-based organisation.
++ Where you get really unclear about membership. Clear expression about the process of becoming a member / falling out of membership. I see lots of co-ops saying "we all know really" but not making it explicit. Something goes wrong 6 months down the line, but nobody knows quite how to deal with it, because nobody knows how to deal with it or quite what happened.
+
+### Employment and rates
+Calverts has the same day rate if you're trained. Members are employed, not freelancers. £20ph NET. Charge-out rate £95ph.
+
+The economy of hours is incredibly important. In this enterprise, which needs all of us, whatever role we are carrying out, we consider every role to be important in the proper functioning of the co-op. Our fees to clients are pretty much average for a firm of our size. Our overall wage bill is roughly average. But the way that we divide it is radical.
+
+Depends on employed vs self-employed vs mixture.
+
+To some degree, you learn these things as you go along. You develop the skills as you need them.
+
+Co-ops are people organising around the question of work.
+
+You can build flexibility into an employment model. A zero-hours contract model can be helpful, in order to retain flexibility but still be employed and gain all the benefits of being employed.
+
+### Work first or structure first?
+Noga: would you agree that we focus on getting some work done together and worry about structures once we know that we have that working?
+Sion: It's a bit chicken and egg - whether you get governance & pay defined before work or start work first. But you don't want to embark on some work together without the same vision in mind.
```

Comment:
If you want this to appear on a new line you need to have a line break between these.